modules/core/test/v2/unit/coins/eos.ts

```diff
@@ -212,4 +215,133 @@ describe('EOS:', function () {
       explainedTx.memo.should.equal('1');
     });
   });
+
+  describe('Transaction Verification', function () {
```

Comment:
Where are the tests for the failure cases? This seems like only a happy path test.

Reply:
added failure cases